app/models/menu.ts

```diff
@@ -242,16 +242,11 @@ if (process.platform === 'darwin') {
     ]);
 }
 
-/**
- * Only allow the Developer Tools to be triggered in a development build.
- */
-if (process.env.ENV === 'development') {
-    let viewSubmenu = viewMenu.submenu;
-    if (viewSubmenu && viewSubmenu instanceof Array) {
-        viewSubmenu.push({
-            role: 'toggledevtools',
-        });
-    }
+let viewSubmenu = viewMenu.submenu;
```

Comment:
I think we should address #167 before merging this in. Currently from a packaged build I can't open examples that have a custom image because it can't find the image file.